File: August/Week4/Day5/fizzBuzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function (n) {
    let fizzBuzzRes = [];
    for (let i = 1; i <= n; i++) {
        if (i % 3 === 0 && i % 5 !== 0) fizzBuzzRes.push('Fizz');
        // return 'Fizz'
        else if (i % 3 !== 0 && i % 5 === 0) fizzBuzzRes.push('Buzz');
        // return 'Buzz'//
        else if (i % 3 === 0 && i % 5 === 0) fizzBuzzRes.push('FizzBuzz');
        // return 'FizzBuzz
        else fizzBuzzRes.push(i.toString()); //return i.toString()
    }

    return fizzBuzzRes;
};

File: August/Week4/Day5/fizzBuzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        // System.out.print(n);
        List<String> res = new ArrayList();
        for(int i=1; i<=n;i++) {
            if(i%3 == 0 && i%5 == 0) {
                res.add("FizzBuzz");
            }
            else if(i%3 != 0 && i%5 == 0) {
                res.add("Buzz");
            }
            else if(i%3 ==0 && i%5 != 0) {
                res.add("Fizz");
            }
            else {
                res.add(Integer.toString(i));
            }
        }
        
        return res;
    }
}